colossalai/inference/batch_bucket.py

```diff
@@ -42,6 +42,9 @@ def __init__(
         self.device = device or get_current_device()
         self.dtype = dtype
 
+        self._use_spec_dec = False
+        self._ver_num = None
```

Comment:
What is `ver_num`? version number?

Reply:
Verification number, or num_to_verify: it's the number of tokens generated by the small/drafter model to verify in the context of large/main model as a verifier. Might revise to name to something like num_to_verify?

Reply:
Renamed.